src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,52 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATION_WITH_DOT =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public static String getSalaryInfo(String[] names, String[] data,
+                                       String dateFrom, String dateTo) {
+        String nameOfEmployee = "";
+        LocalDate date;
+        LocalDate dateFromParsed;
+        LocalDate dateToParsed;
+        int timesGotSalary;
+        int salary;
```

Comment:
try declaring the salary variable inside the loop where you iterate over `names` - it will represent total salary for each person